deploy/lib/server_config.rb

```diff
@@ -2685,8 +2685,9 @@ def ServerConfig.properties(prop_file_location = @@path)
     properties = ServerConfig.load_properties(default_properties_file, "ml.")
     properties.merge!(ServerConfig.load_properties(properties_file, "ml."))
 
+    #Look for optional shared_config, if it is set grab the properties from path relative to the root of the roxy project
     if properties['ml.shared_config'] then
```

Comment:
We could certainly add a file existence test, but that could cause issues with figuring out *if*  the shared config file is working. In other words, if the shared config file doesn't exist, and we're trying to use it, and the code silently jumps over the missing file, how do you know you've made a mistake? Sorry for the run on sentence.